src/components/account/Quota.vue

```diff
@@ -0,0 +1,109 @@
+<!--
+  - SPDX-FileCopyrightText: 2023 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-FileCopyrightText: 2024 STRATO AG
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<section class="section">
+		<h2>{{ t('simplesettings', 'Storage usage') }}</h2>
+		<div class="quota">
+			<NcProgressBar
+				v-if="hasLimitedSpace"
+				size="medium"
+				:value="usageRelative"
+				:color="barColor"
+				:style="{'background-color': backgroundColor}" />
+			<div class="quota-info">
+				<p>
+					<span v-if="hasLimitedSpace" :style="quotaPillStyles(barColor)" />
+					<span v-html="quotaUsedText" />
+				</p>
+				<p v-if="hasLimitedSpace">
+					<span :style="quotaPillStyles(backgroundColor)" />
+					<span v-html="quotaFreeText" />
+				</p>
+			</div>
+		</div>
+	</section>
+</template>
+
+<script>
+import { loadState } from '@nextcloud/initial-state'
+import NcProgressBar from '@nextcloud/vue/dist/Components/NcProgressBar.js'
+
+/** SYNC to be kept in sync with `lib/public/Files/FileInfo.php` */
+const SPACE_UNLIMITED = -3
+
+const barColor = null
+const backgroundColor = null
```

Comment:
we could remove those and set the default as part of the `data()` function

```js
data() {
		return {
			usageRelative,
			barColor: null,
			backgroundColor: null,
			hasLimitedSpace,
			freeSpace,
			usage,
		}
	},
```

Reply:
✅